app/views/parent_objects/show.html.erb

```diff
@@ -60,6 +60,22 @@
   <%= @parent_object.authoritative_metadata_source.display_name %>
 </p>
 
+<p>
+  <strong>Child Count:</strong>
+  <%= @parent_object.child_object_ids.count %>
+</p>
+
+<div class="child-objects">
+  <strong>Children:</strong>  
+    <div class="child-list">
+      <ol>
+        <% @parent_object.child_objects.map{|child| %>
+          <li><%= link_to( "#{child.label || 'no label'} (#{child.width}, #{child.height})", "#{child_objects_path}/#{child.oid}") %></li>
```

Comment:
Can we include the caption here as well, since it sounds like, although it's not in the legacy data, it will be in the Goobi data?

Reply:
Good idea. Added caption.